server/routerlicious/packages/lambdas/src/scribe/lambdaFactory.ts

```diff
@@ -176,38 +193,47 @@ export class ScribeLambdaFactory
 				this.enableWholeSummaryUpload,
 			);
 			latestSummary = await summaryReader.readLastSummary();
+			latestSummaryCheckpoint = latestSummary.scribe
+				? JSON.parse(latestSummary.scribe)
+				: undefined;
+			latestDbCheckpoint = (await this.checkpointService.restoreFromCheckpoint(
```

Comment:
Technically this could be `await`ed in parallel with the summary read. Code would be messy though.